bugsnag/src/test/java/com/bugsnag/ConfigurationTest.java

```diff
@@ -95,6 +103,27 @@ public void testInvalidSessionEndpoint() {
         assertEquals("http://sessions.example.com", getDeliveryEndpoint(config.sessionDelivery));
     }
 
+    @Test
+    public void testSerializeObject() throws SerializationException {
```

Comment:
This doesn't feel like the right level of test - the unit tests are testing that configuration is set right, rather than that what is provided does the right thing. So you could pass your own implementation of `Serializer` in that simply writes "foo" to the output stream and then validate that this is what you receive back; but we shouldn't compare JSON outputs.